spec/spec_helper_acceptance.rb

```diff
@@ -93,6 +94,15 @@ def get_instances(name)
     end.flatten
   end
 
+  def get_instance(name)
```

Comment:
these methods duplicate those currently found in the individual tests. ie. https://github.com/puppetlabs/puppetlabs-aws/blob/master/spec/acceptance/instance_spec.rb#L13-L17.

Happy for this to move here, but we should change the other specs as well.